src/Lib/Util/Password.hs

```diff
@@ -1,30 +1,45 @@
 {-# LANGUAGE FlexibleContexts #-}
 
+{-# LANGUAGE DeriveGeneric    #-}
```

Comment:
I think both these extensions can be put into default extensions for whole project.